lib/model/model.go

```diff
@@ -2457,6 +2486,48 @@ func (m *Model) CommitConfiguration(from, to config.Configuration) bool {
 	return true
 }
 
+// checkFolderRunningLocked returns nil if the folder is up and running and a
+// descriptive error if not.
+// Need to hold (read) lock on m.fmut when calling this.
+func (m *Model) checkFolderRunningLocked(folder string) error {
+	_, ok := m.folderRunners[folder]
+	if !ok {
```

Comment:
could reverse pretty much all of these conditions and get a flatter indentation with early returns